pkgs/development/tools/rust/cargo-ndk/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
```

Comment:
Maybe also remove pkg-config? Please try.

Reply:
It does work, as there's no native dependencies anymore :)